source/common/runtime/runtime_protos.h

```diff
@@ -11,6 +11,32 @@
 namespace Envoy {
 namespace Runtime {
 
+// Helper class for runtime-derived uint32.
+class UInt32 : Logger::Loggable<Logger::Id::runtime> {
+public:
+  UInt32(const envoy::config::core::v3::RuntimeUInt32& uint32_proto, Runtime::Loader& runtime)
+      : runtime_key_(uint32_proto.runtime_key()), default_value_(uint32_proto.default_value()),
+        runtime_(runtime) {}
+
+  const std::string& runtimeKey() const { return runtime_key_; }
+
+  uint32_t value() const {
+    uint64_t raw_value = runtime_.snapshot().getInteger(runtime_key_, default_value_);
+    if (raw_value > std::numeric_limits<uint32_t>::max()) {
+      ENVOY_LOG_EVERY_POW_2(warn,
+                            "value:{} of {} is larger than uint32 max, return default instead",
```

Comment:
Sorry one more-- can you mention in the message that it's a parsed runtime value and swap "return" for "returning". 

It'll be good to go after that.

Reply:
Sure.